conda/recipes/ucx-py/meta.yaml

```diff
@@ -41,10 +44,12 @@ test:
     - ucp
 
 about:
-  home: https://rapids.ai/
-  license: BSD-3-Clause
-  license_family: BSD
-  license_file: ../../../LICENSE
-  summary: Python bindings for UCX
-  dev_url: https://github.com/rapidsai/ucx-py
-  doc_url: https://ucx-py.readthedocs.io
+  home: {{ data.get("project", {}).get("urls", {}).get("Homepage", "") }}
```

Comment:
Should the homepage point at the docs site? I don't really mind one way or the other, WDYT?

Reply:
pyproject.toml supports storing arbitrary URLs in the table, and the repo is already stored under the Source key as well as the Homepage key. As long as all the relevant pages are somewhere under URLs, I don't have a strong preference as to which we designate as the "Homepage". I do think that we'd want to be consistent in what the conda page refers to as the homepage and what the PyPI index will show as the homepage, so any change should be made to pyproject.toml's Homepage so that it automatically propagates to here.

Reply:
Yep agree with Vyas

Only think I'd add is we now also include `doc_url` to cover the documentation page. So that is referenced as well